daemon/mgr/container_diskquota.go

```diff
@@ -0,0 +1,117 @@
+package mgr
```

Comment:
I am thinking if we could move this package out of package mgr, since it is not related to manager, but an underlying work to finish storage setting or something similar.